infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -322,6 +322,15 @@ public void loadedInitialStateResource(
     }
   }
 
+  public void errorIncompatibleInitialState(final UInt64 epoch) {
+    log.error(
+        "Cannot start with provided initial state for the epoch {}, "
+            + "checkpoint occurred on the empty slot, which is not yet supported. \n"
+            + "If you use remote checkpoint source, please, "
+            + "retry an epoch later (6 minutes for most networks).",
+        epoch);
```

Comment:
let's remove the space before \n

"If you are using"?
"please wait next epoch and retry?"